doc/source/conf.py

```diff
@@ -78,49 +80,46 @@ def __getattr__(cls, name):
 # built documents.
 #
 
-import satpy.version as current_version
-
+version = __version__.split('+')[0]
```

Comment:
What is the end result of this? Wouldn't this mean that new documentation would use the previous tag? Meaning we could have 0.10 documentation using the 0.9 version number?

Reply:
no, the version is based on the last tag, so if you have a tag like v9.8.7, you'll get a `__version__` like `9.8.7+f8e228c`, which is the full release in sphinx terms and we build the sphinx version as only the first part of this (before the `+`)

Reply:
Right, so if you have `9.8.7+f8e228c` that means you may have documentation that is for code *after* version 9.8.7. So to have readthedocs publish the "latest" docs would mean that a user would see a site for version "9.8.7" but it could include documentation for post-9.8.7 code.

Maybe this is acceptable. Maybe readthedocs doesn't build/publish from `master`? If so, fine by me.

Reply:
@mraspaud any other comments on this issue?